core/templates/pages/exploration-editor-page/statistics-tab/issues/cyclic-transitions-issue.component.ts

```diff
@@ -41,9 +41,9 @@ angular.module('oppia').component('cyclicTransitionsIssue', {
       };
 
       var issueResolved = false;
-      $scope.resolveIssue = function() {
+      $scope.resolveIssueAsync = function() {
```

Comment:
@the-known-anon, this function doesn't return any promise, so it's not expected to change the name of this function. Can you please undo this change? 
[Also, undo the changes of this function names in other places (like in test and HTML files)]

Reply:
Hello,
sorry for the late reply, but in the file "https://github.com/oppia/oppia/blob/develop/core/templates/services/playthrough-issues-backend-api.service.ts", _resolveIssue_ does return a Promise. Could you have a look at it?

Reply:
@the-known-anon I agree that the PlaythroughIssuesBackendApiService.resolveIssue does return a Promise. But I don't think the `$scope.resolveIssue` returns a promise (I think it doesn't return anything.), can you please check and update the PR accordingly? :)